modules/oe_theme_helper/tests/src/Unit/TwigExtensionTest.php

```diff
@@ -300,4 +303,110 @@ public function toEclIconProvider(): array {
     ];
   }
 
+  /**
+   * Tests trimming of textfield.
+   *
+   * @param mixed $textfield
+   *   The string, object or render array.
+   * @param array $context
+   *   The context array.
+   * @param mixed $expected_result
+   *   The expected result.
+   *
+   * @covers ::trimTextfield
+   * @dataProvider trimTextfieldProvider
+   */
+  public function testTrimTextfield($textfield, array $context, $expected_result): void {
+    $context['input_data'] = $textfield;
```

Comment:
The function needs the context for the length, not for the input data. Please set the length in the context from the data provider.

Reply:
Could you explain more.